lib/services/elasticsearch.js

```diff
@@ -1204,11 +1215,8 @@ function assertNoRouting(esRequest) {
  * @param {object} esRequest
  * @throws
  */
-function assertWellFormedRefresh(version, esRequest) {
+function assertWellFormedRefresh(esRequest) {
   if (esRequest.hasOwnProperty('refresh')) {
-    if (compareVersions(version, '5.0') < 0) {
-      throw new BadRequestError(`Refresh parameter is not supported by the version "${version}" of Elasticsearch API`);
-    }
     if (esRequest.refresh !== 'wait_for' && esRequest.refresh !== 'false' && esRequest.refresh !== false) {
       throw new BadRequestError('Refresh parameter only supports the value "wait_for" or false');
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Merge this if statement with the nested one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ACollapsibleIfStatements)